checkov/common/util/prompt.py

```diff
@@ -0,0 +1,273 @@
+from checkov.common.checks.base_check_registry import BaseCheckRegistry
+from checkov.common.util.banner import banner as checkov_banner
+from pathlib import Path
+from jinja2 import Environment, PackageLoader, select_autoescape
+from jinja2 import Template
+import click
+import jinja2
+import os
+import yaml
+import importlib
+
+CHECKOV_ROOT_DIRECTORY = os.path.join(".", "checkov")
+TEMPLATE_DIRECTORY = os.path.join(os.path.dirname(__file__), "templates")
+
+
+class Prompt(object):
+    ACTIONS = ["add"]
+    CHECK_CLASS = ["terraform"]
+    CATEGORIES = ["application_security", "backup_and_recovery" "convention", "encryption",
+                  "general_security", "iam", "kubernetes", "logging", "networking", "secrets"]
+    TERRAFORM_OBJECT_TYPE = ["data", "provider", "resource"]
+    PROVIDERS = ["azure", "aws", "gcp"]
+
+    PROMPTS = {
+        "action": {
+            "text": 'What action would you like to take?',
+            "typ": click.Choice(ACTIONS),
+            "default": "add",
+            "sub_prompts": [
+                {
+                    "title": {
+                        "text": 'Enter the title of your new check (without a .py)',
+                        "typ": str,
+                        "default": "MyNewTest"
+                    },
+                    "category": {
+                        "text": 'Select a category for this check',
+                        "typ": click.Choice(CATEGORIES),
+                        "default": "iam"
+                    },
+                    "desc": {
+                        "text": 'Describe what this check does',
+                        "typ": str,
+                        "default": "Ensure that X does Y..."
+                    },
+                    "check_class": {
+                        "prompt_if": 'add',
+                        "text": 'What what kind of check would you like to add?',
```

Comment:
```suggestion
                        "text": 'What kind of check would you like to add?',
```

Reply:
Not sure how I did this or how you caught it. Nice one!